bridging-tutorial-website/docs/getting-started.mdx

```diff
@@ -214,15 +214,7 @@ yarn android
 
 "New Architecture" is the future and that's why it will be used throughout this tutorial. Guides will showcase how to write code that supports both old and new arch. However, because new arch setups are more complex, you'll benefit from testing the native code you wrote with TurboModules and Fabric and the backward compatibility layer will make it work on the old architecture.
 
-To enable "New Architecture" on Android:
-- go to `android/gradle.properties` in your IDE
-- locate `newArchEnabled` variable and set it to `true`
-- run `./android/gradlew -p android clean` from your app's root directory - this will invoke clean Gradle task on your Android project
-
-To enable "New Architecture" on iOS:
-- go to `ios/Podfile` in your IDE
-- add `ENV['RCT_NEW_ARCH_ENABLED'] = "1"` after `require_relative '../node_modules/@react-native-community/cli-platform-ios/native_modules'` - this will set the `RCT_NEW_ARCH_ENABLED` to `1` on each `pod install` run, so you'll not need to set that env before running installing the Pods
-- run `npx pod-install` to install "New Architecture" Pods
+Since the new architecture is developing dynamically and the instructions for this step are changing often, follow the official React Native [New-architecture-app-intro](https://reactnative.dev/docs/new-architecture-app-intro).
```

Comment:
I could't find the "require_relative..." in the podspec and it looks like it changed in RN 0.72. I assumed it could be hard to keep this step up to date so I changed it this way. What do you think about it?

Reply:
Awesome idea! It's better to refer to the core docs